lib/sgtn-client/api/translation.rb

```diff
@@ -54,15 +54,15 @@ def self.getString_f(component, key, args, locale, *optionals)
          return s
       end
 
-      def self.getStrings(component, locale)
+      def self.getStrings(component, locale=SgtnClient::Config.configurations.default)
         SgtnClient.logger.debug "[Translation][getStrings]component=#{component}, locale=#{locale}"
         items = get_cs(component, locale)
         default = SgtnClient::Config.configurations.default
         if items.nil? || items["messages"] == nil
           items = {}
           s = SgtnClient::Source.getSources(component, default)
           if s.nil?
-            SgtnClient.logger.error "[Translation]can't find the component '" + component + "' in source path!"
+            SgtnClient.logger.debug "[Translation][getStrings]can't find component=#{component}, locale=#{locale}!!!"
```

Comment:
This is for a different, higher-priority issue: https://github.com/vmware/singleton/issues/1493
Next time, please keep the PR as small as possible.